pgo/cmd/auth.go

```diff
@@ -66,8 +66,19 @@ func UserHomeDir() string {
 }
 
 func parseCredentials(dat string) msgs.BasicAuthCredentials {
+	lines := strings.Split(strings.TrimSpace(dat), "\n")
+	if (len(lines) != 1){
+		log.Debug("found more than one line in credentials file")
```

Comment:
I'd update to reflect the current reality of the check (e.g. "expected one and only one line - found %d")